src/ray/core_worker/core_worker.cc

```diff
@@ -1099,11 +1099,16 @@ void CoreWorker::Disconnect(
 
   opencensus::stats::StatsExporter::ExportNow();
   if (connected_) {
-    RAY_LOG(INFO) << "Disconnecting to the raylet.";
+    RAY_LOG(INFO) << "Sending disconnect message to the local raylet.";
     connected_ = false;
     if (local_raylet_client_) {
-      RAY_IGNORE_EXPR(local_raylet_client_->Disconnect(
-          exit_type, exit_detail, creation_task_exception_pb_bytes));
+      Status status = local_raylet_client_->Disconnect(
+          exit_type, exit_detail, creation_task_exception_pb_bytes);
+      if (status.ok()) {
+        RAY_LOG(INFO) << "Disconnected from the local raylet.";
+      } else {
+        RAY_LOG(WARNING) << "Failed to disconnect from the local raylet.";
```

Comment:
Should we print status in the logging message?

Reply:
added